features/bookmarks/bookmark_privacy.feature

```diff
@@ -101,32 +102,33 @@ Feature: Private bookmarks
       And I go to the bookmarks page
     Then I should not see "Secret Masterpiece"
       And I should see "Public Masterpiece"
-      And I should not see "bookmarker"
+      And I should not see "avid_bookmarker"
       And I should see "otheruser"
-    When I go to bookmarker's bookmarks page
+    When I go to avid_bookmarker's bookmarks page
     Then I should not see "Secret Masterpiece"
       And I should not see "Public Masterpiece"
     When I go to the works page
     Then I should not see "Secret Masterpiece"
       And I should see "Public Masterpiece"
       And I should not see "Bookmarks: 2"
-      # CACHING (perform_caching: true)
-      # And I should see "Bookmarks: 1"
+      And I should see "Bookmarks: 1"
     When I view the work "Public Masterpiece"
     Then I should not see "Bookmarks:2"
       And I should see "Bookmarks:1"
     When I follow "1"
     Then I should see "List of Bookmarks"
       And I should see "Public Masterpiece"
       And I should see "otheruser"
-      And I should not see "bookmarker"
+      And I should not see "avid_bookmarker"
 
-      # Private bookmarks should not show on tag's page
+    # Private bookmarks should not show on tag's page
     When I go to the bookmarks tagged "Stargate SG-1"
     Then I should not see "Secret Masterpiece"
       And I should see "Public Masterpiece"
-      And I should not see "bookmarker"
+      And I should not see "avid_bookmarker"
```

Comment:
Because of the separation of the filters/queries in the sidebar, the lowercase string "bookmarker" now appears on all bookmark listings when the new search is enabled. So I renamed the user in this test to make it easier to check whether their username is associated with any publicly visible bookmarks.